pycm/pycm_param.py

```diff
@@ -23,6 +23,8 @@
 MATRIX_FORMAT_ERROR = "Input confusion matrix format error"
 MAPPING_FORMAT_ERROR = "Mapping format error"
 MAPPING_CLASS_NAME_ERROR = "Mapping class names error"
+SEABORN_PLOT_LIBRARY_ERROR = "Error in importing seaborn module. Please install it : pip install seaborn"
+MATPLOTLIB_PLOT_LIBRARY_ERROR = "Error in importing matplotlib module. Please install it : pip install matplotlib"
```

Comment:
Please change it to:
"Error in importing seaborn module. Please install it using this command: pip install matplotlib"